### DISCUSSION TITLE
Importing Sheets-Times Not Importing

### DISCUSSION THREAD

**Discussion by tsvmitch** [cpvalente/ontime#766] [Q&A]
When I import a cue sheet with the start/end/duration, they all show as 00:00:00 in OnTime. I tried adding the start/end times and duration in Onetime and then pushing back to the google sheet to see what happened. I noticed there is now a leading apostrophe in each of the cells. Is there something I'm missing, or is this how I need to have my show callers set up their run-of-shows to use the sync feature?

**Comment by cpvalente** [comment]
Hi, thank you for reaching out.

We are not aware of issues with import of time. Would you mind sending us a copy of your spreadsheet? 

If you are not comfortable with sending the spreadsheet, just an example that reproduces the issue is good enough. You can send it to mail@getontime.no

**Comment by cpvalente** [comment]
We have spoken on email, but I will reply here for users who might be looking for a response.
In version 2, it seems that we have found issues in two scenarios which match the user file
- Importing time fields in AM/ PM such as: `00:09:00 AM`
- Importing spreadsheets which do not include the Project header (Title, description ...)

I hope both of this will be resolved in v3.
For users finding the same issue, adding the project header and converting your fields to 24h times should fix the imports
